### DISCUSSION TITLE
Is there a way to use a string rather than the cli?

### DISCUSSION THREAD

**Discussion by sroussey** [timostamm/protobuf-ts#606] [Q&A] [answered]
Is there a way to use a string rather than the cli? For example:

```ts
const GetRequest = proto3.makeMessageType('foo.GetRequest', `
syntax="proto3";
message GetRequest {
	int32 a = 1;
}
`);
```

Thanks!

**Comment by jcready** [comment, marked as answer]
No: https://github.com/timostamm/protobuf-ts/issues/317#issuecomment-1134387340

**Comment by sroussey** [comment]
That is a bummer. I was thinking about implementing something for Chrome DevTools to inspect such messages.
